dialogflow_v2/gapic/agents_client.py

```diff
@@ -13,13 +13,15 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
 """Accesses the google.cloud.dialogflow.v2 Agents API."""
 
 import functools
 import pkg_resources
 import warnings
 
 from google.oauth2 import service_account
+import google.api_core.client_options
```

Comment:
Add client_options to v2

Reply:
So client_options are basically a way to set the endpoint directly?